primitives/src/v4/mod.rs

```diff
@@ -819,121 +815,14 @@ pub struct ParathreadEntry {
 	pub retries: u32,
 }
 
-/// An Assignemnt for a paras going to produce a paras block.
-#[derive(Clone, Encode, Decode, PartialEq, TypeInfo, RuntimeDebug)]
```

Comment:
I remember these are actually not part of v4 at the moment and you are undoing previous work done in [alex/parathreads_review](https://github.com/paritytech/polkadot/tree/alex/parathreads_review), right?

Reply:
Exactly! At the end of the day, we'll need to make sure v4 is just as it is on `master`. Which I think it is here but will confirm later down the road when we have the final feature branch with a diff to `master`.